pkgs/development/python-modules/opt-einsum/test-value_error.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/opt_einsum/tests/test_input.py b/opt_einsum/tests/test_input.py
+index e5c1677..54ea059 100644
+--- a/opt_einsum/tests/test_input.py
++++ b/opt_einsum/tests/test_input.py
+@@ -34,7 +34,7 @@ def test_type_errors():
+         contract("", 0, out='test')
+ 
+     # order parameter must be a valid order
+-    with pytest.raises(TypeError):
++    with pytest.raises(ValueError):
```

Comment:
I don't feel like this is a nixpkgs-specific issue, is there a way to upstream it?

Reply:
I have created an issue upstream: https://github.com/dgasmith/opt_einsum/issues/147